cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureEncryptionResources.java

```diff
@@ -46,6 +46,10 @@
 
     private static final Pattern ENCRYPTION_KEY_URL_VAULT_NAME = Pattern.compile("https://([^.]+)\\.vault.*");
 
+    private static final Pattern ENCRYPTION_VAULT_URI = Pattern.compile("(.*)\\/keys\\/.*");
+
+    private static final Pattern ENCRYPTION_KEY_NAME = Pattern.compile("https:.*\\/keys\\/([^.]+)\\/.*");
+
```

Comment:
Suggestion to consider : Is it possible to use single regex like `"(https://([^.]+).vault.[^/]+)/keys/([^/]+)/.*"` where group 1 will be vault uri, 2 will be vault name and 3 will be key name?

Also `\\/` I guess these double escaping is not needed, it should work without those.